DNN Platform/Modules/ResourceManager/Components/SearchController.cs

```diff
@@ -34,26 +34,14 @@ public IList<IFileInfo> SearchFolderContent(int moduleId, IFolderInfo folder, bo
 
             search = (search ?? string.Empty).Trim();
 
-            // Lucene does not support wildcards as the beginning of the search
```

Comment:
Did something change with Lucene's support for wildcards?  What's the behavior if you search with a wildcard at the beginning of your search?

Reply:
@bdukes I couldn't for the life of me figure out why this was even there.  It works as expected without this unnecessary startsWith trimming.